### ISSUE TITLE
Strapi Admin Blank Screen -- TypeError: Cannot read property 'forEach' of undefined

### ISSUE THREAD

**Issue by MakingStuffs** [strapi/strapi#7241]
**Describe the bug**
When I go to the admin panel, after logging in, I am confronted with a blank screen. 

I followed this guide for deployment: https://strapi.io/documentation/3.0.0-beta.x/deployment/heroku.html

**Steps to reproduce the behavior**

Deploy the app to heroku using postgreSQL. Then navigate to admin panel and login/create user.

**Expected behavior**
The admin panel to appear.

**Screenshots**

**System**

- Node.js version: 10.x
- NPM version: 6.0.0
- Strapi version: 3.0.5
- Database: postgreSQL
- Operating system: Ubuntu

**Additional context**
I'm not really sure what the problem is but I am able to give more information if it is required. 

The server index page loads correctly and says the server is running. 

I have initiated the build command -- My start script is: `NODE_ENV=production npm run build && strapi start`. 

I added `admin: {url: "/admin"}` to the server.js file also.

The only information I can see which may help someone more experienced is the console logs the following:

```
An error has occured
main.bd56623c.chunk.js:1 --------------------
main.bd56623c.chunk.js:1 TypeError: Cannot read property 'forEach' of undefined
    at main.bd56623c.chunk.js:1
    at Array.reduce (<anonymous>)
    at h (main.bd56623c.chunk.js:1)
    at Za (main.bd56623c.chunk.js:1)
    at gc (main.bd56623c.chunk.js:1)
    at sl (main.bd56623c.chunk.js:1)
    at cl (main.bd56623c.chunk.js:1)
    at $c (main.bd56623c.chunk.js:1)
    at main.bd56623c.chunk.js:1
    at t.unstable_runWithPriority (main.bd56623c.chunk.js:1)
main.bd56623c.chunk.js:1 Here is some infos

conmponentStack: "
    in h
    in div
    in styled.div
    in u
    in div
    in styled.div
    in ie
    in n
    in A
    in injectIntl(Connect(n))
    in t
    in d
    in t
    in div
    in styled.div
    in div
    in styled.div
    in Unknown
    in l
    in x
    in A
    in t
    in t
    in IntlProvider
    in n
    in A
    in Ti
    in l"
```

Thank you in advance. I really am stumped and need to fix this issue

**Comment by anya92** [comment]
I have the same problem with my app deployed on Google App Engine.

**Comment by lauriejim** [comment]
@soupette any insight here?

**Comment by MakingStuffs** [comment]
I ended up fixing the issue by cloning the following repo into an empty folder: https://github.com/strapi/strapi-starter-blog

Then I ran the develop script and completed creating an admin. 

Following that I followed the instructions here: https://strapi.io/documentation/v3.x/deployment/heroku.html

Then, I deleted the current data from within the admin panel and copied over the api and components folder from my previous setup. Obviously I now need to re populate both databases, but at least it works. 

This makes me think there was something wrong with how the project was setup before. I was using the following guide for reference: https://strapi.io/blog/build-a-static-blog-with-gatsby-and-strapi
